templates/zerver/integrations/google-calendar.md

```diff
@@ -1,72 +1,64 @@
-    to your `~/` directory.
+{start_tabs}
 
-1.  Next, install the latest Google API Client for Python by following the
-    instructions on the
-    [Google website](https://developers.google.com/api-client-library/python/start/installation).
+1.  Download and install our
+    [Python bindings and example scripts](/api/installation-instructions).
 
-1.  In Zulip, go to your click on the cog in the top right corner, and
-    then clicking on **Personal settings**.
+1.  Install the requirements for the integration script with:
 
-1.  Click on the tab that’s labeled **Account & privacy** and click on
-    **Manage your API key**. Enter your password if prompted, and
-    download the `zuliprc` file. Save this file as `.zuliprc` to your `~/`
-    directory.
+    `pip install /usr/local/share/zulip/integrations/google/requirements.txt`
 
-    ![Download zuliprc file](/static/images/integrations/google/calendar/001.png)
+1.  Follow [Google's instructions][google-quickstart] to get a
+    `client_secret` file, and save it as `client_secret.json` in your
+    `~/` directory.
 
-1.  Run the `get-google-credentials` with this command:
+1.  Install the latest Google API Client for Python by following [Google's
+    instructions][google-api-client].
+
+1.  In Zulip, click the **gear** (<i class="fa fa-cog"></i>) icon at the top
+    right, select **Personal settings**. Select the **Account & privacy**
+    tab, and click **Manage your API key**. Enter your password if prompted,
+    download the `zuliprc` file, and save it as `.zuliprc` in your`~/`
+    directory.
 
-        python /usr/local/share/zulip/integrations/google/get-google-credentials
+1.  To authorize your Zulip bot to view your Calendar, run the script:
 
-1.  It should open up a browser and ask you for certain permissions. Give
-    Zulip access, and move on to the next step. If it doesn’t open a
-    browser, follow the instructions in the terminal window.
+    `python /usr/local/share/zulip/integrations/google/get-google-credentials`
 
-1.  Now, all that’s left to do is to run the `gcal-bot` script, in the
-    same directory as the `get-google-credentials` script, with the
-    necessary parameters:
+    Authorize access in the browser window that opens. If no browser opens,
+    follow the terminal prompts.
 
-        python /usr/local/share/zulip/integrations/google/gcal-bot --user foo@zulip.com
+1.  In the same directory, start the Zulip bot with the command:
 
-    The `--user` flag specifies the user to send the reminder to.
+    `python /usr/local/share/zulip/integrations/google/google-calendar --user <your user email for Zulip>`
 
-1.  Don’t close the terminal window with the bot running (you can use
-    `screen` if needed). You will only get reminders if the bot is still
-    running.
+    Use the [optional flags](#configuration-options) to configure the bot.
 
-{!congrats.md!}
+1.  You will get reminders as long as the terminal session with the bot
```

Comment:
The word "only" was removed from "You will only get reminders".
The `screen` part was kept in a tip, now has been bundled with this instruction.